pkgs/stdenv/freebsd/always-patchelf.sh

```diff
@@ -0,0 +1,5 @@
+fixupOutputHooks+=(_FreeBSDPatchelfShrink)
+
+_FreeBSDPatchelfShrink() {
+    find $prefix -type f | xargs -n1 patchelf --shrink-rpath &>/dev/null || true
```

Comment:
Maybe only executable files need to be patchelf'd?

Reply:
not necessarily - shared objects are often -x and need to be patchelfed.